notebooks/get-wiki-articles.py

```diff
@@ -46,7 +46,7 @@ def main():
             title = wikipedia.page(state).title.lower().replace(" ", "_")
             content = (
                 wikipedia.page(state).summary
-                if args.only_summary
+                if args.short_summary
                 else wikipedia.page(state).content
             )
```

Comment:
```suggestion
            )
            content = content.strip()
```

Reply:
Hi @Hemanthkumar2112! Thank you for the contribution! I noticed some trailing whitespace in the files when they get saved off. I recommend stripping the extra whitespace in order to keep the data as clean as possible.